satellite/console/consoleweb/utils.go

```diff
@@ -67,16 +67,15 @@ func getQuery(w http.ResponseWriter, req *http.Request) (query graphqlJSON, err
 
 // queryPOST retrieves graphql query from POST request
 func queryPOST(w http.ResponseWriter, req *http.Request) (query graphqlJSON, err error) {
+	limitedReader := http.MaxBytesReader(w, req.Body, ContentLengthLimit.Int64())
```

Comment:
This isn't being closed when the content-type is some different. Use `defer` to close things, it helps to avoid these bugs.

Reply:
Ah nevermind, actually with HTTP handler, the Server automatically closes the Body. But would be still nice to use defer here.